packages/calcite-components/src/components/color-picker/color-picker.tsx

```diff
@@ -1034,6 +1034,8 @@ export class ColorPicker
         numberingSystem={this.numberingSystem}
         onCalciteInputNumberChange={this.handleChannelChange}
         onCalciteInputNumberInput={this.handleChannelInput}
+        onFocus={this.handleInputFocus}
+        onInput={this.handleChannelInputChange}
```

Comment:
For consistency, can you use the calcite-specific events? Also, the event name doesn't match the event (e.g., named after the `change` event, but it's listening to `input`). Applies to the hex-input changes.